src/main/java/org/jenkinsci/plugins/workflow/support/DefaultStepContext.java

```diff
@@ -225,4 +225,11 @@ public Set<String> getSensitiveVariables() {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getStepDisplayFunctionName() throws IOException {
```

Comment:
As per https://github.com/jenkinsci/workflow-step-api-plugin/pull/204#discussion_r1965996465 I would suggest adding a clause near https://github.com/jenkinsci/workflow-support-plugin/blob/4696ddb8b47bb834ab6a6df00a73b18cb6623118/src/main/java/org/jenkinsci/plugins/workflow/support/DefaultStepContext.java#L84-L85

Reply:
(My warning about #252 you noted https://github.com/jenkinsci/workflow-support-plugin/pull/306#issuecomment-2674488371 was because I predicted a merge conflict there.)

Reply:
using like this now.